src/app/components/topics/change-log.tsx

```diff
@@ -3,6 +3,18 @@ import React from 'react';
 import envConfig from '../../../envConfig';
 
 const submissionChangesTable = [
+  ['08/31/22', `Submissions API Infrastructure Update in Developer Preview`, `
+  <ul>
+    <li>
+      Developer Preview has been updated to utilize a modern API framework.
+    </li>
+    <li>
+      Production will be updated prior to Submissions Opening in Jan 2023.
+    </li>
+    <li>
+      See <a href='${envConfig.cmsGithubIo}/qpp-developer-preview-docs/error-codes' target='_blank' rel='noopener noreferrer'>Error Codes</a> for addition details.
```

Comment:
I'm guessing should read: `for additional details.`

Reply:
Good catch! Resolved.